dyno_driver.go

```diff
@@ -88,13 +88,14 @@ func lookupLocalIP() (string, error) {
 	}
 
 	for _, addr := range addrs {
-		ip, ok := addr.(*net.IPNet)
-		if !ok && ip.IP.IsLoopback() {
+		ipnet, ok := addr.(*net.IPNet)
+		if !ok && ipnet.IP.IsLoopback() {
 			continue
 		}
 
-		if v4 := ip.IP.To4(); v4 != nil {
-			return v4.String(), nil
```

Comment:
According to the doc, this should not contain the subnet already: http://golang.org/pkg/net/#IP.String